pymobiledevice3/cli/developer.py

```diff
@@ -97,6 +97,15 @@ def applist(lockdown, color):
         print_json(apps, colored=color)
 
 
+@dvt.command('send-signal', cls=Command)
+@click.argument('pid', type=click.INT)
+@click.argument('signal', type=click.INT)
+def send_signal(lockdown, pid, signal):
```

Comment:
let's name the command simply `signal`

Reply:
sure. I will change the function as well